contrib/completion/bash/docker

```diff
@@ -5395,6 +5395,23 @@ _docker_wait() {
 	_docker_container_wait
 }
 
+BUILDX_PLUGIN_PATH=$(docker info --format '{{range .ClientInfo.Plugins}}{{if eq .Name "buildx"}}{{.Path}}{{end}}{{end}}')
```

Comment:
Maybe for a follow-up, but I think we should try to make it generic for plugins. We should call `docker info` only once and process the result, depending the plugin name.
It might be tricky though, so maybe for a follow-up 🤔

Reply:
Thanks for your feedback. I will make it more generic to support all plugins. Keep you posted.